.circleci/config.yml

```diff
@@ -23,5 +23,5 @@ workflows:
       - test:
           matrix:
             parameters:
-              python: ["3.6", "3.7", "3.8", "3.9"]
-              django: ["22", "30", "31"]
+              python: ["3.8", "3.9", "3.10"]
+              django: ["3", "4"]
```

Comment:
given there are subtle differences, shouldn't we explicitly test 3.2, 4.0, etc.

Reply:
Maybe I shouldn't have simplified this yeah... It's now testing with 3.2 and 4.0 though, see the tox config: https://github.com/mozilla/django-post-request-task/pull/30/files#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7R25-R26

Reply:
I could go back to more explicit names. Would you prefer this ? :

```diff
diff --git a/.circleci/config.yml b/.circleci/config.yml
index 89b75ba..a3c9202 100644
--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -24,4 +24,4 @@ workflows:
           matrix:
             parameters:
               python: ["3.8", "3.9", "3.10"]
-              django: ["3", "4"]
+              django: ["32", "40"]
diff --git a/tox.ini b/tox.ini
index fc2e512..6956dbd 100644
--- a/tox.ini
+++ b/tox.ini
@@ -8,11 +8,11 @@ deps =
     pytest
     flake8
     coverage
-    django3: Django>=3.2,<4.0
-    django4: Django>=4.0,<5.0
+    django32: Django>=3.2,<4.0
+    django40: Django>=4.0,<5.0
 commands =
     pip install -e .
     make test
 
 [tox]
-envlist = {py38,py39,py310}-django{3,4}
+envlist = {py38,py39,py310}-django{32,40}
```

It's the same matrix of tests in the end.